ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor.hs

```diff
@@ -645,7 +664,7 @@ peerSelectionGovernorLoop tracer
       --   Check the definition site for more details, but in short, when the
       --   node changes to 'TooOld' state it will go through a purging phase which
       --   the 'waitForTheSystemToQuiesce' monitoring action will wait for.
-      <> Monitor.monitorLedgerStateJudgement actions st
+      <> Monitor.monitorLedgerStateJudgement actions readBigLedgerSnapshot ledgerPeersCtx st
```

Comment:
See here `actions` would already have everything you need.

Reply:
Done

Reply:
@bolt12 did you mean `readBigLedgerSnapshot` and `ledgerPeersCtx`, or sth else?